assets/reader-activation/auth.scss

```diff
@@ -219,41 +269,48 @@
 			align-items: center;
 			background: white;
 			border: none;
-			border-radius: 0;
+			border-radius: 3px;
 			box-shadow: none;
 			color: inherit;
 			cursor: pointer;
 			display: flex;
 			font-size: inherit;
-			height: 36px;
+			height: 40px;
 			justify-content: center;
-			margin: 0;
-			padding: 6px;
+			margin: 8px;
+			padding: 0;
 			position: absolute;
 			right: 0;
 			top: 0;
-			width: 36px;
+			width: 40px;
+
+			@media screen and ( min-width: 600px ) {
+				height: 30px;
+				margin: 2px;
+				width: 30px;
+			}
+
+			@media screen and ( min-width: 782px ) {
+				height: 48px;
+				margin: 8px;
+				width: 48px;
+			}
 
 			svg {
 				fill: currentcolor;
 				flex: 0 0 24px;
 			}
 
-			&:active,
-			&:hover {
-				opacity: 0.6;
-			}
-
 			&:focus {
-				outline: 1px solid;
-				outline-offset: -1px;
+				outline: 1.5px solid #111;
+				outline-offset: 2px;
 			}
 		}
 
 		&__terms-text {
-			font-size: 0.8em;
-			margin: 0.5rem 0;
-			color: #757575;
+			font-size: 0.625em;
```

Comment:
This text is now minuscule! I feel like it's a bit _too_ small at 10px.

Reply:
My bad, I wanted 11px. Fixed here: https://github.com/Automattic/newspack-plugin/pull/1870/commits/ec3611c059f10a0fb9bafcf2dc428b1cfca086dd

Reply:
It's still pretty small—if it were up to me I'd make it 13px to match the size of the other "supplemental" text like "Lost your password?" But that's totally subjective, so I won't block the PR for this.